go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/couchbaselabs/gocbconnstr
+module github.com/couchbaselabs/gocbconnstr/v2
```

Comment:
I think this is all that's required (plus a release) but genuinely can't remember...